operators/grpc_operators/client/entity_client.hpp

```diff
@@ -0,0 +1,133 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CLIENT_ENTITY_CLIENT_HPP
+#define CLIENT_ENTITY_CLIENT_HPP
+
+#include <chrono>
+#include <thread>
+
+#include <grpcpp/grpcpp.h>
+#include <gxf/std/tensor.hpp>
+
+#include "../common/asynchronous_condition_queue.hpp"
+#include "../common/conditional_variable_queue.hpp"
+#include "../common/tensor_proto.hpp"
+#include "../generated/holoscan.grpc.pb.h"
+#include "../generated/holoscan.pb.h"
```

Comment:
Something of a nitpick, but shouldn't we be directly including these directories in the target?
```suggestion
#include <asynchronous_condition_queue.hpp>
#include <conditional_variable_queue.hpp>
#include <tensor_proto.hpp>
#include <holoscan.grpc.pb.h>
#include <holoscan.pb.h>
```

Reply:
Updated. Thank you.